test/app.test.js

```diff
@@ -372,10 +410,51 @@ describe("app", function() {
             it('should accept valid options', function () {
                 return tester
                     .setup.user.state('No_SA_ID_Gender_Entry')
+                    .setup.user.answers(completed_answers)
                     .input('1')
                     .check.reply.content(/Thank you! Your report has been submitted/)
                     .run();
             });
         });
+
+        describe('Submit_Case', function () {
+            it('should make a valid Ona API call', function () {
+                return tester
+                    .setup.user.state('Submit_Case')
+                    .setup.user.answers(completed_answers)
+                    .input('1')
+                    .check.interaction({
+                        state: 'Facility_Code_Entry',
+                        reply: /Welcome! To report a malaria case, please enter your facility code./
+                    })
+                    .check(function (api) {
+                        var http_sent = _.where(api.http.requests, {
+                            url: 'http://ona.io/api/v1/submission'
+                        })[0];
```

Comment:
Get the HTTP request that was made to Ona for which we're returning a fixture as a mock response.